### PR TITLE
Modified UI for try it. Fixes for try-it.

### PR DESCRIPTION
**Pull request by hemikak**

### CODE REVIEW

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/rest/TryItService.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ * Copyright (c) 2016-2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (srsharon) [inline modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/rest/TryItService.java]: This should be just 2016

File: modules/web/src/plugins/try-it/views/http-client.scss [hunk 2]
```diff
@@ -225,63 +219,185 @@ hr{
...
+            .react-autosuggest__container {
+                .inputContainer {
+                    height: 33px;
+                    padding: 6px;
+                    border: none;
+                    border-bottom: 1px solid #3b3b3b;
+                    input[type=text]{
+                        color: #fff;
+                        background-color: #333;
+                        border: 0px solid #333;
```
Comment (srsharon) [inline modules/web/src/plugins/try-it/views/http-client.scss]: remove px from 0

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/rest/TryItService.java (hunk 1 above)
Comment (hemikak) [reply modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/rest/TryItService.java]: Fixed in https://github.com/ballerinalang/composer/pull/4354

File: modules/web/src/plugins/try-it/views/http-client.scss (hunk 2 above)
Comment (hemikak) [reply modules/web/src/plugins/try-it/views/http-client.scss]: Fixed in https://github.com/ballerinalang/composer/pull/4354

**Pull request merged**